cmd/job-runtime-analyzer/main.go

```diff
@@ -1,20 +1,39 @@
 package main
```

Comment:
I don't think anyone uses this tool anymore. I believe that all this info is already in BigQuery anyways. I guess I would rather see this removed than changed so heavily, but we could get confirmation from the rest of the team first.

Reply:
Nice, let's dig into it and if nobody is using, we remove.